addons/TweenSuite/TweenAnimation.gd

```diff
@@ -7,6 +7,7 @@ extends Resource
 class_name TweenAnimation
 
 var steps: Array
+var parameters: Dictionary;
```

Comment:
```suggestion
var parameters: Dictionary
```
You can also add `@export`, so it's editable in the inspector and serialized.

Reply:
I'm on Godot 4.3 right now so I didn't have the feature yet, but I know it's coming.
I also don't know what's your philosophy concerning versions, do you prefer to target as many versions as possible or only the last one ?

As for the export, I'm not sure if it would be useful ? If people want to serialize values they can do it in your editor already. Maybe if they want a default value for the parameter and then sometimes modify it in code ?

Reply:
> I also don't know what's your philosophy concerning versions, do you prefer to target as many versions as possible or only the last one ?

I prefer targeting the latest version if there is a feature that I find useful (in 4.3 it was e.g. property picker). However if the plugin is incompatible with the current stable version, I have to wait before I can update it in AssetLib. Afterwards the users can download the older version of the addon from GitHub.

You can leave the Dictionary untyped, I might update it myself in the future.

> Maybe if they want a default value for the parameter and then sometimes modify it in code ?

Yes.
But it's fine without export too.